### ISSUE TITLE
useEvent: Typescript warning, Window !== RefTyped

### ISSUE THREAD

**Issue by blowsie** [pikax/vue-composable#49]
```
useEvent(window, 'keypress', ({ keyCode }) => {
//...
})
```
>Argument type Window is not assignable to parameter type RefTyped

**Comment by pikax** [comment]
quick fix is to cast as any

```ts
useEvent(window as any, 'keypress', ({ keyCode }) => {
// ...
})
```

I will prepare a fix to go out

**Comment by blowsie** [comment]
Well, im not actually using typescript in my project, but Webstorm is understand the types and throwing warnings. 
Thanks!

**Comment by pikax** [comment]
published [0.2.2](https://www.npmjs.com/package/vue-composable)
